src/rf/apps/core/migrations/0022_auto_20151019_1041.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
```

Comment:
I like to rename these autogenerated files to something more useful and descriptive.